frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -236,6 +237,17 @@ if Device:isKobo() then
     }
 end
 
+if Device:isKindle() and PowerD:hasHallSensor() then
+    common_settings.cover_events = {
+        text = _("Cover events"),
+        help_text = _([[Enable or disable the hall effect sensor.
```

Comment:
"The Hall-effect principle is named for physicist Edwin Hall. In 1879 he discovered that when a conductor or semiconductor with current flowing in one direction was introduced perpendicular to a magnetic field a voltage could be measured at right angles to the current path."
May be capitilize "hall"?

Reply:
Incidentally, I've been considering buying a [Gulikit KingKong 2](https://www.gulikit.com/productinfo/737791.html), with Hall effect sensors.

But over here as I said I'm not sure how much sense it makes to talk about the underlying tech instead of what it does (i.e., detect covers).

Reply:
Also, I understand that the Kindle has it enabled by default ? And that you added that option to allow disabling it because it sucks ?
Then I would just name it `[ ] Disable cover events` (or `[ ] Disable Kindle cover events` if it's really clear to all it's Kindle system stuff outside of us).

Reply:
I meant the capitalization only.